### PR TITLE
Feature/function calling parallel support

### PR DESCRIPTION
**Pull request by Jakob-98**
PR For parallel function calling support

see https://platform.openai.com/docs/guides/function-calling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,6 +152,9 @@ orchestrator.register_all([weatherService.get_current_weather])
 # ...
 ```
 
+#### Parallel function calling
+
+Parallel function calling is a feature supported in [certain models](https://platform.openai.com/docs/guides/function-calling). Calling parralel functions is supported with the orchestrator. For this, use `orchestrator.generate_tools_descriptions()`. See the [orchestrator parallel example](./examples/orchestrator_example_parallel.py) for more details.
```
Comment (arjendev) [inline README.md:157]: typo in "parralel"

File: openai_functools/functions_orchestrator.py [hunk 2]
```diff
@@ -221,3 +228,26 @@ def create_function_descriptions(
             ]
         )
         return [spec.parameters for spec in specs]
+
+    def create_tools_descriptions(
+        self, selected_functions: Optional[List[str]] = None
+    ) -> List[Dict[str, Any]]:
+        """
+        Creates descriptions for the selected functions.
```
Comment (arjendev) [inline openai_functools/functions_orchestrator.py:236]: Open question: the description of this function is very similar to create_function_descriptions. Can we assume that the user knows the difference between function descriptions and tool descriptions? Should we make it explicit in the docstring?

File: README.md (hunk 1 above)
Comment (arjendev) [inline README.md:157]: Should we also add support for parallel function calling when registering instances?

**Review by arjendev — CHANGES_REQUESTED** [review]
Added a few comments, it's looking great though!

File: README.md (hunk 1 above)
Comment (Jakob-98) [reply README.md]: It works already, I added it to the examples

File: openai_functools/functions_orchestrator.py (hunk 2 above)
Comment (Jakob-98) [reply openai_functools/functions_orchestrator.py]: Made the docstrings more explicit

File: README.md (hunk 1 above)
Comment (Jakob-98) [reply README.md]: Thanks for the review. Addressed

**Comment by arjendev** [comment]
Thanks!!

**Review by arjendev — APPROVED** [review]
Thanks

**Pull request merged**